pkgs/by-name/ce/ceph/rocksdb_7_9/default.nix

```
@@ -0,0 +1,140 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  bzip2,
+  lz4,
+  snappy,
+  zlib,
+  zstd,
+  windows,
+  enableJemalloc ? false,
+  jemalloc,
+  enableLiburing ? stdenv.hostPlatform.isLinux,
+  enableLite ? false,
+  enableShared ? !stdenv.hostPlatform.isStatic,
+  sse42Support ? stdenv.hostPlatform.sse4_2Support,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rocksdb";
+  version = "7.9.2";
+
+  src = fetchFromGitHub {
+    owner = "ceph"; # note this is a fork, not the upstream `facebook`
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "rocksdb";
```
pkgs/by-name/ce/ceph/package.nix